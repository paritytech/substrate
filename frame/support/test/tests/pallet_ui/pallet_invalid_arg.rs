#[frame_support::pallet(foo)]
pub mod pallet {}

fn main() {}
