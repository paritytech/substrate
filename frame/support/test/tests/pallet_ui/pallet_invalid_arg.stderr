error: Invalid pallet macro call: unexpected attribute. Macro call must be bare, such as `#[frame_support::pallet]` or `#[pallet]`, or must specify the `dev_mode` attribute, such as `#[frame_support::pallet(dev_mode)]` or #[pallet(dev_mode)].
 --> tests/pallet_ui/pallet_invalid_arg.rs:1:25
  |
1 | #[frame_support::pallet(foo)]
  |                         ^^^
